Sources/TokamakCore/Modifiers/TextFieldStyle.swift

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+public protocol TextFieldStyle {
+  // FIXME: Should properties be internal?
+  var type: String { get }
+  var style: String { get }
+  init()
+}
+
+public struct DefaultTextFieldStyle: TextFieldStyle {
```

Comment:
I agree it would be best to keep those internal, how's about this?

```suggestion
public protocol TextFieldStyle {
}

protocol _TextFieldStyle: TextFieldStyle {
  var type: String { get }
  var style: String { get }
  init()
}

public struct DefaultTextFieldStyle: _TextFieldStyle {
```